app/assets/javascripts/school_administrator_dashboard/dashboard_components/discipline_dashboard/SchoolwideDisciplineIncidents.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import SchoolDisciplineDashboard from './SchoolDisciplineDashboard';
+import DashboardHelpers from '../DashboardHelpers';
+import {latestNoteDateText} from '../../../helpers/latestNoteDateText';
+
+class SchoolWideDisciplineIncidents extends React.Component {
+
+  schoolDisciplineEvents() {
```

Comment:
This data shape is referred to as `totalDisciplineIncidents` in the SchoolDisciplineDashboard. When I was trying to figure out where the data shape came from, I kept searching for `totalDisciplineIncidents`, but it's named `schoolDisciplineEvents()` at the parent level. Do you think it would simplify to use `totalDisciplineIncidents()` up here too?

Reply:
Definitely!